### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_ScreenReader:The labels are not present for the edit box.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9345]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.1.1;#MAS3.3.2;#VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voiceover
Mac OS:10.15.3
MAS References:MAS1.1.1 MAS3.3.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Launch screen reader as Voiceover
8. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.
9. Navigate to the 9 fields using tab under "advanced storage setting button"

**Actual:**
When navigated to all 9 fields under advanced storage setting button there is no label provided for the fields, just a placeholder that is similar for all and announced by a screen reader.
**Except for the first field of every row.**

**Expected:**
Each input field should have a label or instruction and the label should help the user complete the task.

**User Impact:**
Screen reader and keyboard users will not know with respect to what label the data needs to be entered in the field.

**Attachment link for Reference :**
![image](https://user-images.githubusercontent.com/24580570/75322428-b731f800-5898-11ea-8c23-7f7dfa66ebf6.png)

**Comment by sspatil123** [comment]
As "SQL Server Big Data Cluster" feature has been removed
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
